README.md

```diff
@@ -119,7 +116,34 @@ The following prerequisites are required:
+
+Based on this, in order to make CoatRack work with the FIWARE Keyrock, you will need to:
+
+- First, pull and deploy the containers of keyrock via docker (as the Keyrock itself needs to be running in your local machine). To do this, just follow
+the tutorial in https://fiware-idm.readthedocs.io/en/latest/getting_started/index.html
+- Please check that Keyrock is running correctly in your local machine by checking if you can access the Keyrock GUI,
+- If everything is running well, then we need to start by register CoatRack as application in Keyrock, to do this:
+  - Login or Create a user account. By default, you can use the admin@test.com, the credentials are in the environment variables: `IDM_ADMIN_EMAIL`, `IDM_ADMIN_USER` and `IDM_ADMIN_PASS`,
+  ![KeyRock User Login](./spring-boot/admin/src/main/resources/static/images/keyrock/KeyRockLandingPage.png)
+  - After login, go to the application and click in the register button,
+  ![KeyRock register application](./spring-boot/admin/src/main/resources/static/images/keyrock/KeyRockRegisterApplication.png)
+  - Fill in all the fields in the registration form. The Callback URL must be `http:localhost:8080/login/aouth2/code/keyrock` as shown in the figure below. This is important because these values will be in the CoatRack configuration file later,
+  ![KeyRock application form](./spring-boot/admin/src/main/resources/static/images/keyrock/KeyRockApplicationFormFilled.png)
+  - With your application already created (Attention: only the Callback URL needs to be the same, so for example you can define the application name as you wish), the next step will be to add the credentials that are required for CoatRack to access Keyrock to the CoatRack config in the `application-keyrock.yml`.
+  In order to do this configuration, you will need the Oauth2 credentials, so go to the previously created application and click in OAuth2 Credentials, and you should see a ClientId and ClientSecret as shown in the screenshot,
+  ![KeyRock Oauth2 Credentials](./spring-boot/admin/src/main/resources/static/images/keyrock/KeyRockOauth2Credentials.png)
+  - After the Oauth2 credentials were obtained, please copy them and replace in the application-keyrock.yml, as marked in the figure below,
+  ![KeyRock Oauth2 Credentials](./spring-boot/admin/src/main/resources/static/images/keyrock/KeyRockConfig.png)
+ 
+- After all of this (creating the application in KeyRock and added its config to CoatRack), you can now build and run CoatRack, this could be done in 2 ways:
+  - Via the CLI, so after building the jar file for coatrack admin, just run the command ```java -jar -Dspring.profiles.active=keyrock spring-boot/admin/target/coatrack-admin-*.jar```,
+  - Via Docker, by going into `coatrack/docker/docker-compose-deployment/docker-compose.yml`, and change the line that contains ```--spring.profiles.active=github``` to ```--spring.profiles.active=keyrock``` as shown in the screenshot below,
```

Comment:
So, I checked regarding overriding a parameter, and this would make it the script much more complex, as we are using inside of the shell script to run coatrack with docker, the docker-compose command which doesn't work with any spring profile, so we would need to create a code to go to the docker compose file and replace the value for the input of the user.
Based on this, I find it easier that the user itself changes the docker compose file directly

Reply:
I added then a link to the Build with docker section